### DISCUSSION TITLE
Syncing Google Calander and/or iCalander

### DISCUSSION THREAD

**Discussion by shfroyle** [mikeberger/borg_calendar#171] [General]
So, I had to recently switch to PC and I really don't like Microsoft's suite and I just discovered Borg and it is literally everything I've ever wanted in a calander app. But I have a couple issues, not sure if its possible to get help with.

1. Is there a way to sync my Google Calander? I am not a tech wizard by any means, so if you could break it down a little it would be amazing.
2. Same question, but for iCal. This isn't as important because, if nothing else, I can move those from iCal to Google Calander and then sync them.
3. IF I make calander changes on my phone via iCal or Google Calander, will those changes automatically sync and appear in Borg desktop?

Thanks so much!

**Comment by mikeberger** [comment]
1. For google calendar, read this first: https://github.com/mikeberger/borg_calendar/discussions/155

I will add more info to this other discussion when I get a chance. There is a lot of complexity due to how google APIs work. 

2. You can sync BORG with CALDAV servers, and some work very well, but some won't due to how old and primitive the third-party CALDAV library that I use is: https://github.com/mikeberger/borg_calendar/discussions/153

3. Yes - with some limitations. Other calendar programs, like google, outlook, thunderbird calendar, mobile apps ... treat appointments wildly differently from each other even though they all claim to adhere to standards. I'll elaborate more in the google discussion.

I'll add more info to the google discussion ....
